clients/pkg/promtail/targets/docker/target_group.go

```diff
@@ -72,9 +72,9 @@ func (tg *targetGroup) addTarget(id string, discoveredLabels model.LabelSet) err
 		}
 	}
 
-	_, ok := tg.targets[id]
+	t, ok := tg.targets[id]
 	if ok {
-		level.Debug(tg.logger).Log("msg", "ignoring container that is already being scraped", "container", id)
```

Comment:
Does this debug log have no value anywhere?

Reply:
Actually you are right. Let me add it.